### PR TITLE
The lean angle is now applied independently of the function generateR…

### LINKED ISSUE #22: Apply lean angle outside the function generateRobotPoseAlongStripper

### PR DESCRIPTION
**Pull request by Nandite**
@VictorLamoine @KevinBollore 
Fix #22

### CODE REVIEW

**Comment by VictorLamoine** [comment]
:+1: Kevin what do you think?

File: bezier_library/src/bezier_grinding_surfacing.cpp [hunk 1]
```diff
@@ -121,6 +121,13 @@ std::string BezierGrindingSurfacing::generateTrajectory(EigenSTL::vector_Affine3
     grinding_trajectories.push_back(traj);
   }
 
+  // Application of the grinding lean angle
+  for (EigenSTL::vector_Affine3d poses : grinding_trajectories)
+  {
+    for (Eigen::Affine3d pose : poses)
```
Comment (KevinBollore) [inline bezier_library/src/bezier_grinding_surfacing.cpp]: More readable like this?
`for (Eigen::Affine3d pose: poses)`

**Comment by KevinBollore** [comment]
Without my comment about readability, it looks great

File: bezier_library/src/bezier_grinding_surfacing.cpp (hunk 1 above)
Comment (VictorLamoine) [inline bezier_library/src/bezier_grinding_surfacing.cpp]: You mean the space before the semi-colon?
If yes; the ROS formatting expects a space before so it's fine as is.

Comment (KevinBollore) [inline bezier_library/src/bezier_grinding_surfacing.cpp]: So it's ok for me :)

**Pull request merged**